CMakeLists.txt

```diff
@@ -522,7 +533,7 @@ if (BUILD_SHARED_LIBS)
 					LINK_FLAGS "-Wl,--version-script,${CMAKE_CURRENT_BINARY_DIR}/src/${SYMBOL_FILENAME}"
 					)
 			else ()
-				target_link_options (sndfile PRIVATE "LINKER:--version-script,${CMAKE_CURRENT_BINARY_DIR}/src/${SYMBOL_FILENAME}")
+				target_link_options (sndfile PRIVATE)
```

Comment:
Please don't touch. It will break ABI for all Unix-based OSes.

Reply:
If I keep the line as-is, I get the following error:
ld: unknown option: --version-script